config/webpack/loaders/devSiteExample.js

```diff
@@ -11,6 +12,31 @@ const loader = async function loader() {
   const exampleSource = this.resourcePath;
   const parsedResourcePath = path.parse(exampleSource);
 
+  let cssFileName;
+  let exampleCssSource;
+
+  /**
+   * Use a try catch block in order to synchronously capture the file name for any attached css files
+   */
+  try {
+  // eslint-disable-next-line import/order
+    const lineReader = require('readline').createInterface({
+      input: require('fs').createReadStream(exampleSource),
```

Comment:
Let's remove these global imports and import them at the top of the file

Reply:
Removed when I refactored devSiteExample : [a74d9a](https://github.com/cerner/terra-dev-site/pull/281/commits/a74dc9a3d2b15e5aa59680d0ff59033e916a0f89).